mica-core/src/main/java/org/obiba/mica/micaConfig/domain/CollectionDatasetConfig.java

```diff
@@ -10,5 +10,5 @@
 
 package org.obiba.mica.micaConfig.domain;
 
-public class StudyDatasetConfig extends DatasetConfig {
+public class CollectionDatasetConfig extends DatasetConfig {
```

Comment:
I add this to the ticket : I didn't make this in this pull request because it needs a migration script, and I'll make another pull request for this (all config objects)